app/src/lib/rebase.ts

```diff
@@ -66,3 +70,30 @@ export function initializeRebaseFlowForConflictedRepository(
 export function formatRebaseValue(value: number) {
   return Math.round(clamp(value, 0, 1) * 100) / 100
 }
+
+/**
+ * Check application state to see whether the action applied to the current
+ * branch should be a force push
+ */
+export function isCurrentBranchForcePush(
```

Comment:
glad to see this extracted into a helper function!

could we also use this helper function here? https://github.com/desktop/desktop/blob/639afe70d2dc271b0d9dd9183e507adae6346f3a/app/src/ui/changes/no-changes.tsx#L315-L318

Reply:
Sure can!

Reply:
Addressed in 12ea76d48b4473e06ce7a63d216c9fcf955f8f91